ui/pages/Popup.tsx

```diff
@@ -200,8 +210,16 @@ export function Main(): ReactElement {
                     >
                       <TopMenu />
                     </div>
-                    {/* @ts-expect-error TODO: fix the typing when the feature works */}
                     <Switch location={transformedLocation}>
+                      {
+                        // If an account doesn't exist, display onboarding
```

Comment:
```suggestion
                        // If there are no existing accounts, display onboarding
```

Because if there are accounts loaded then it should switch to another address, not display onboarding

Reply:
Good catch, I'll update the comment